relays.yaml

```
@@ -57,4 +57,5 @@ relays:
   - 'wss://nostr-relay.nonce.academy'
   - 'wss://nostr.rewardsbunny.com'
   - 'wss://nostr.slothy.win'
+  - 'wss://relay.nostropolis.xyz/websocket
```

You're missing a single quote:
```suggestion
  - 'wss://relay.nostropolis.xyz/websocket'
```

---

wow.. sorry about that. thanks for the fix